src/test/java/tests/application/browser/BrowserTests.java

```diff
@@ -0,0 +1,51 @@
+package tests.application.browser;
+
+import aquality.selenium.core.application.IApplication;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class BrowserTests {
+    @Test
+    public void testShouldBePossibleToGetApplication() {
+        IApplication browser = AqualityServices.getServiceProvider().getInstance(IApplication.class);
+        Assert.assertNotNull(browser, "should be possible to resolve app instance from aquality services");
+        Assert.assertEquals(AqualityServices.getApplication(), browser,
+                "should return the same instance of app");
+    }
+
+    @Test
+    public void testShouldBePossibleToGetDriver() {
+        Assert.assertNotNull(AqualityServices.getApplication().getDriver(),
+                "should be possible get driver from the application");
+    }
+
+    @Test
+    public void testShouldBePossibleToSetImplicitWait() {
+        AqualityServices.getApplication().setImplicitWaitTimeout(1, TimeUnit.SECONDS);
```

Comment:
Would it better to wrap this into try\catch and use asser.fail in catch block?
I think test should have assert and some message in fail case.

Reply:
fixed